synapse/config/tls.py

```diff
@@ -165,31 +155,3 @@ def generate_files(self, config):
                 cert_pem = crypto.dump_certificate(crypto.FILETYPE_PEM, cert)
 
                 certificate_file.write(cert_pem)
-
-        if not self.path_exists(tls_dh_params_path):
-            if GENERATE_DH_PARAMS:
```

Comment:
GENERATE_DH_PARAMS is still defined (line 25)

Reply:
oops. fixed.